lib/bundle/brew_installer.rb

```diff
@@ -31,6 +32,10 @@ def install_change_state!
       end
     end
 
+    def start_service?
+      @start_service
+    end
+
     def restart_service?
       # Restart if `restart_service: :always`, or if the formula was installed or upgraded
       @restart_service && (@restart_service.to_s != 'changed' || changed?)
```

Comment:
Did some local testing and this behaviour is not correct for this PR; it'll only return `true` if `:restart_service => true` and it's been upgraded. This method should mirror `start_service?` above and be just `@restart_service` and a new method should be added called `restart_service_needed?` which uses the `changed` logic. Make sense? Sorry; I know I said the last change was the last one but I tested this locally and it doesn't work properly for `restart` otherwise.